app/frontend/stylesheets/limber/screen.scss

```diff
@@ -399,14 +398,14 @@ dl#samples-information {
 .state-badge.passed {
   @extend.bg-success;
 }
-.state-badge.qc_completed {
-  @extend.bg-success;
+.state-badge.qc_complete {
+  @extend.bg-primary;
 }
 .state-badge.cancelled {
-  @extend .list-group-item-danger;
+  @extend .bg-danger;
```

Comment:
Can `cancelled` colour be different from failed ? For example, `.bg-warning` rather than `.bg-danger` ?